src/components/Notifications/TimeConfig.tsx

```diff
@@ -73,71 +90,67 @@ export const TimeConfigComponent: React.FunctionComponent = () => {
                                     </HelperTextItem>
                                 </HelperText>
                             </StackItem>
                         </Stack>
                         <br></br>
                         <Split>
                             <SplitItem isFilled>
-                                {timePref ?
-                                    <Stack hasGutter>
-                                        <StackItem>
-                                            <Radio
-                                                isChecked={ radioSelect && !showCustomSelect }
-                                                onChange={ handleRadioSelect }
-                                                value={ timePref }
-                                                id='settings-time-config'
-                                                label='Default time'
-                                                description='12:00 UTC'
-                                                name='radio-select'>
-                                            </Radio>
-                                        </StackItem>
-                                        <StackItem>
-                                            <Radio
-                                                isChecked={ radioSelect && showCustomSelect }
-                                                value={ timePref }
-                                                onChange={ handleCustomRadioSelect }
-                                                id='settings-time-config'
-                                                label='Custom time'
-                                                description='Choose specific time and time zone'
-                                                name='radio-select'>
-                                            </Radio>
+                                <Stack hasGutter>
+                                    <StackItem>
+                                        <Radio
+                                            isChecked={ radioSelect && !showCustomSelect }
+                                            onChange={ handleRadioSelect }
+                                            id='settings-time-config'
+                                            label='Default time'
+                                            description='00:00 UTC'
+                                            name='radio-select'>
+                                        </Radio>
+                                    </StackItem>
+                                    <StackItem>
+                                        <Radio
+                                            isChecked={ radioSelect && showCustomSelect }
+                                            onChange={ handleCustomRadioSelect }
+                                            id='settings-time-config'
```

Comment:
We need to use a different id here.
```suggestion
                                            id='settings-time-config-custom'
```